spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/zuul/filters/post/SendErrorFilter.java

```diff
@@ -76,6 +77,8 @@ public Object run() {
 				request.setAttribute("javax.servlet.error.message", message);
 			}
 
+			//unwrap request if needed, otherwise tomcat hangs in endless loop
```

Comment:
@spencergibb  Source code for the `ServletRequestWrapper` being used at that point?